src/glassflow/client.py

```diff
@@ -27,18 +29,26 @@ def __init__(self, organization_id: str = None) -> None:
         self.glassflow_config = GlassFlowConfig(rclient)
         self.organization_id = organization_id
 
-    def pipeline_client(self, space_id: str, pipeline_id: str,
-                        pipeline_access_token: str) -> PipelineClient:
+    def pipeline_client(self,
+                        pipeline_id: str = None,
+                        pipeline_access_token: str = None,
+                        space_id: str = None) -> PipelineClient:
```

Comment:
Should we set the type hint to `Optional[str]` if we set it to `None` as default?

Reply:
thanks, made the change and updated the pr